### ISSUE TITLE
Just a question

### ISSUE THREAD

**Issue by kaimokene** [piratjsk/Rby#6]
If i were to use more fish on a public server can it move over to Rby without issues when it comes out? Or should i wait until rby comes out before using more fish on my server.

**Comment by piratjsk** [comment]
Yes. I plan to make process of configs migration completely automatic and all of MoreFish commands will still work in Rby.

Also, it looks like elsiff [may be returning](https://github.com/elsiff/more-fish/commit/1f77d1648075489b2803dfd6a06d8aefb01daf23), so maybe you won't even have to move to Rby (although you'll be able to as I plan to develop it no matter if other alternatives will exist).
